capa/loader.py

```diff
@@ -176,6 +176,8 @@ def get_extractor(
       UnsupportedArchError
       UnsupportedOSError
     """
+    console = Console(stderr=True, quiet=disable_progress)
```

Comment:
`stderr=True` is used to redirect the spinner banner to stderr, so that users can redirect capa's output.

Reply:
this is a  good comment that should go in the code, so future readers will understand this choice.